micro-infra-spring-base/src/test/groovy/com/ofg/infrastructure/web/resttemplate/fluent/UrlUtilsSpec.groovy

```diff
@@ -0,0 +1,62 @@
+package com.ofg.infrastructure.web.resttemplate.fluent
+
+import spock.lang.Specification
+import spock.lang.Unroll
+
+
+class UrlUtilsSpec extends Specification {
+
+    def "should add parameter to query string" () {
+        given:
+            Map<String, String> parameters = new HashMap<>();
+            parameters.put("firstParam", "firstParamValue")
+            parameters.put("secondParam", "secondParamValue")
+        when:
+            
+            URI result = UrlUtils.addQueryParametersToUri(new URI(""), parameters)
+        then:
+            result.toString() == "?firstParam=firstParamValue&secondParam=secondParamValue"
+    }
+
+    def "should not add 'null' string to null parameter in query string" () {
+        given:
+            Map<String, String> parameters = new HashMap<>();
+            parameters.put("firstParam", null)
```

Comment:
`Map<String, String> parameters = [firstParam:null, secondParam:'']`

the same applies to others